frontend/components/transaction/BidTransaction.vue

```diff
@@ -13,36 +13,45 @@
-            :is-loading="isConnecting"
-            :wallet-address="walletAddress"
-            :is-explanations-shown="isExplanationsShown"
-            @connectWallet="$emit('connect')"
-            @disconnectWallet="$emit('disconnect')"
-        />
-        <DepositBlock
-            class="mb-6 lg:mb-0"
-            :disabled="!auctionTransaction.isActive || !isConnected"
-            :is-loading="isDepositingOrWithdrawing"
-            :is-explanations-shown="isExplanationsShown"
-            :transaction-bid-amount="transactionBidAmount"
-            :wallet-dai="walletDai"
-            :wallet-vat-dai="walletVatDai"
-            @manageVat="$emit('manageVat')"
-        />
-        <AuthorisationBlock
-            class="mb-6 lg:mb-0"
-            :disabled="!isEnoughDeposited || !auctionTransaction.isActive || !isConnected"
-            :auction-transaction="auctionTransaction"
-            :is-loading="isAuthorizing"
-            :is-wallet-authorized="isWalletAuthorized"
-            :is-collateral-authorised="isCollateralAuthorised"
-            :is-explanations-shown="isExplanationsShown"
-            @authorizeWallet="$emit('authorizeWallet')"
-            @authorizeCollateral="$emit('authorizeCollateral', auctionTransaction.collateralType)"
-        />
+        <div class="mb-4">
+            <WalletConnectionCheckPanel
+                :wallet-address="walletAddress"
+                :disabled="!auctionTransaction.isActive || auctionTransaction.isFinished"
+                :is-loading="isConnecting"
+                :is-explanations-shown="isExplanationsShown"
+                @connectWallet="$emit('connect')"
+                @disconnectWallet="$emit('disconnect')"
+                @update:isCorrect="isConnected = $event"
+            />
+            <WalletVatDaiBalanceCheckPanel
+                :wallet-dai="walletDai"
+                :wallet-vat-dai="walletVatDai"
+                :transaction-bid-amount="transactionBidAmount"
+                :is-loading="isDepositingOrWithdrawing"
+                :disabled="!isConnected"
+                @manageVat="$emit('manageVat')"
+                @update:isCorrect="isEnoughDeposited = $event"
+            />
+            <WalletAuthorizationCheckPanel
+                :is-wallet-authorized="isWalletAuthorized"
+                :wallet-address="walletAddress"
+                :disabled="!isEnoughDeposited"
+                :is-loading="isAuthorizingWallet"
+                :is-explanations-shown="isExplanationsShown"
+                @authorizeWallet="$emit('authorizeWallet')"
+            />
+            <CollateralAuthorizationCheckPanel
+                :collateral-type="auctionTransaction.collateralType"
+                :is-collateral-authorized="isCollateralAuthorized"
+                :auth-transaction-fee-e-t-h="auctionTransaction.authTransactionFeeETH"
+                :wallet-address="walletAddress"
+                :is-wallet-authorized="isWalletAuthorized"
```

Comment:
This prop shouldn't be part of the `CollateralAuthorizationCheckPanel` as far as I remember